### ISSUE TITLE
Add ARIA attributes to contact us form

### ISSUE THREAD

**Issue by Avinash905** [SurajPratap10/Imagine_AI#581]
**Is your feature request related to a problem? Please describe.**
Currently, our contact us form lacks ARIA attributes, which can impede the ability of screen readers and assistive technologies to properly interpret and navigate these forms. 

**Describe the solution you'd like**

- ARIA roles: "form" for forms, "region" for form containers, and "heading" for form headings.
- ARIA labels and descriptions: "aria-label" and "aria-describedby" to provide concise and descriptive information.
- ARIA states and properties: "aria-required" for required fields, "aria-invalid" for invalid fields, "aria-autocomplete" for easier form filling, and "aria-errormessage" for associating error messages.

**Comment by Avinash905** [comment]
Please assign me this issue so that I can work on this issue under **GSSOC'23**.
